src/System.Management.Automation/engine/ErrorPackage.cs

```diff
@@ -1071,13 +1074,7 @@ public virtual void GetObjectData(SerializationInfo info, StreamingContext conte
         /// Is this instance serialized.
         /// </summary>
         /// <value></value>
```

Comment:
Is the tag mandatory? Can we remove it?

Reply:
I think it's just automatically added by VS when adding comment XML. I'll remove it. :)